app/src/main/res/layout/all_topics.xml

```diff
@@ -22,7 +22,7 @@
       android:layout_marginEnd="@dimen/all_topics_text_margin_end"
       android:layout_marginBottom="@dimen/all_topics_text_margin_bottom"
       android:fontFamily="sans-serif-medium"
-      android:text="@string/all_topics"
```

Comment:
Can all_topics be removed now, or is it still being used?

Reply:
can be removed, thanks -- removing this